src/components/Container/Container.spec.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import assert from 'assert';
+import { common } from '../../util/generic-tests';
+
+import Container from './Container';
+
+describe('Container', () => {
+	common(Container);
+
+	describe('header', () => {
+		it('should exists', () => {
+			const wrapper = shallow(<Container><Container.Header>Header</Container.Header><Container.Content>Content</Container.Content></Container>);
+			assert.equal(wrapper.find('.lucid-Container-header').length, 1);
+		});
+
+		it('should not exists', () => {
+			const wrapper = shallow(<Container><Container.Content>Content</Container.Content></Container>);
+			assert.equal(wrapper.find('.lucid-Container-header').length, 0);
+		});
+	});
+
+	describe('title', () => {
+		it('should exists', () => {
+			const wrapper = shallow(<Container title={'Title'}><Container.Content>Content</Container.Content></Container>);
+			const header = wrapper.find('.lucid-Container-header');
+			assert.equal(header.length, 1);
+			assert.equal(header.text(), 'Title');
+		});
+
+		it('should not exists', () => {
+			const wrapper = shallow(<Container><Container.Content>Content</Container.Content></Container>);
+			assert.equal(wrapper.find('.lucid-Container-header').length, 0);
+		});
+	});
+
+	describe('content', () => {
+		it('should exists', () => {
```

Comment:
Prefer these to be a bit more descriptive. When this and the next test run in the terminal I'm going to see

```
content
  should exists
  should not exists
```

That's confusing to me. :)